test/badge.js

```diff
@@ -245,18 +245,23 @@ export async function downloadPdfFromBackpack(driver) {
 
     const dropdownButtons = await driver.findElements(By.css(
         'button[role="menuitem"]'));
-    const pdfExportButton = dropdownButtons[1];
+    const pdfExportButton = dropdownButtons[2];
     await pdfExportButton.click();
 
     const htmlEmbed = await driver.wait(until.elementLocated(
         By.css('embed[src^="blob:http"]')), defaultWait);
-    await driver.switchTo().frame(htmlEmbed);
+    // To avoid `No Such Frame` error
+    await driver.wait(until.ableToSwitchToFrame(htmlEmbed), defaultWait);
```

Comment:
The actual frame switching is missing here. Also, I thought this wasn't necessary anymore...

Reply:
Yes, it worked. The error raised when adding switching frame `NoSuchFrameError: no such frame`